test/docTest.js.html

```diff
@@ -423,12 +448,35 @@
             assert.deepEqual($('a').first(), document.activeElement);
           });
 
+          it('should insert attribute into an element passing a key and value', function() {
+            var element = $('#single-attr');
+            var returnElement = element.attr('target', '_blank');
+            assert.equal(element.attr('target'), '_blank');
+            assert.equal(element, returnElement);
+          });
+
+          it('should insert multiple attributes into an element passing an object', function() {
+            var element = $('#multiple-attr');
+            var returnElement = element.attr({'href': '#test', 'target': '_blank'});
+            assert.equal(element.attr('href'), '#test');
+            assert.equal(element.attr('target'), '_blank');
+            assert.equal(element, returnElement);
+          });
+
           it('should remove an attribute from an element', function() {
             var element = $('#remove-my-href');
             element.removeAttr('href');
             assert.notEqual(element.attr('href'), '#not-removed');
           });
 
+          it('should remove an attribute from an element with multiple attributes', function() {
+            var element = $('#multiple-attr');
+            var returnElement = element.attr({'href': '#test', 'target': '_blank'});
```

Comment:
What about adding a third attribute just to guarantee it remains there after removing the other two?